.github/workflows/release-nightly.yml

```diff
@@ -0,0 +1,46 @@
+name: Release Nightly
+
+on:
+  push: 
+    branches: 
+      - main
+      - v*.*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  publish_doc:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "refs/heads/${{ matrix.branch }}"
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: 24.0
+          elixir-version: 1.13.4
```

Comment:
I'm not sure that I should compile Elixir from source or use this is better. (Just use this for quick prototype but it can be change).

Reply:
You need to compile from source, which is why I think using the CI workflow that already builds docs may be better. :)